### ISSUE TITLE
build fails on missing ews.partial

### ISSUE THREAD

**Issue by MarcusCalidus** [ewsjs/xhr#10]
I get a strange error upon including @ews/xhr.

Using 

- node 13.8.0 
- npm 6.13.7.
- Typescript 3.5.3
- @ewsjs/xhr 1.4.2

Build fails on missing ews.partial. 

```
2 import { IXHROptions, IXHRApi, IXHRProgress } from "./ews.partial";
                                                     ~~~~~~~~~~~~~~~

node_modules/@ewsjs/xhr/dist/cookieProvider.d.ts:2:29 - error TS2307: Cannot find module './ews.partial'.

2 import { IXHROptions } from "./ews.partial";
                              ~~~~~~~~~~~~~~~

node_modules/@ewsjs/xhr/dist/index.d.ts:1:10 - error TS2305: Module '"../../../../../../../../../Users/marco.warm/Documents/Development/intranet/node_modules/@ewsjs/xhr/dist/ntlmAuthXhrApi"' has no exported member 'ntlmAuthXhrApi'.

1 export { ntlmAuthXhrApi } from "./ntlmAuthXhrApi";
           ~~~~~~~~~~~~~~

node_modules/@ewsjs/xhr/dist/IProvider.d.ts:1:29 - error TS2307: Cannot find module './ews.partial'.

1 import { IXHROptions } from "./ews.partial";
                              ~~~~~~~~~~~~~~~

node_modules/@ewsjs/xhr/dist/ntlmProvider.d.ts:2:29 - error TS2307: Cannot find module './ews.partial'.

2 import { IXHROptions } from "./ews.partial";
                              ~~~~~~~~~~~~~~~

node_modules/@ewsjs/xhr/dist/xhrApi.d.ts:3:52 - error TS2307: Cannot find module './ews.partial'.

3 import { IXHROptions, IXHRApi, IXHRProgress } from "./ews.partial";
```
                                  
this is my tsconfig.json

```
{
  "compilerOptions": {
    "declaration": false,
    "emitDecoratorMetadata": true,
    "experimentalDecorators": true,
    "lib": [
      "es6",
      "dom"
    ],
    "mapRoot": "../dist/api",
    "module": "commonjs",
    "moduleResolution": "node",
    "outDir": "../dist/api",
    "sourceMap": true,
    "target": "es6",
    "typeRoots": [
      "../node_modules/@types"
    ]
  },
  "typedocOptions": {
    "ignoreCompilerErrors": true,
    "mode": "modules",
    "out": "../doc/api",
    "exclude": "**./*.{js*,docx,md,sh,pg,ico,html,*css}",
    "allowJs": false,
    "module": "commonjs"
  }
}



```

**Comment by MarcusCalidus** [comment]
Version 1.4.1 works though...

**Comment by Sigurdlex** [comment]
@gautamsi any updates on this problem?

**Comment by gautamsi** [comment]
let me try with node 13.

**Comment by Sigurdlex** [comment]
I guess the reason of this error is that file ews.partial.d.ts exists in src folder, but not in dist folder.

**Comment by schadock29** [comment]
Hello, 
are you planning to release the fix ?

**Comment by gautamsi** [comment]
published `1.4.3`
